src/main/java/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait.java

```diff
@@ -90,11 +90,8 @@ public void setSkipNotifications(final boolean skipNotifications) {
 
     @Override
     protected void decorateContext(final SCMSourceContext<?, ?> context) {
-        if (context instanceof GitHubSCMSourceContext) {
-            GitHubSCMSourceContext githubContext = (GitHubSCMSourceContext) context;
-            if (!githubContext.notificationsDisabled()) {
-                githubContext.withNotificationsDisabled(isSkipNotifications());
-            }
+        if (isSkipNotifications()) {
+            ((GitHubSCMSourceContext)context).withNotificationsDisabled(true);
```

Comment:
Is it safe to remove the `instanceof` check?

Reply:
Perhaps this makes it safe <https://github.com/jenkinsci/github-checks-plugin/blob/5fc35572bfecf4364a24277547a5e5970dde0ebb/src/main/java/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait.java#L118-L121>

Reply:
I added, just to make it safer.